pkg/sentry/monitoring/metrics.go

```diff
@@ -44,36 +44,35 @@ var (
 )
 
 // CertSignRequestReceived counts when CSR received.
-func CertSignRequestReceived() {
-	stats.Record(context.Background(), csrReceivedTotal.M(1))
```

Comment:
For things like telemetry and monitoring, IMHO it's more correct to use a background context than one specific to the request, since the recording should happen regardless of what happens to the request.

Reply:
I agree with you but in this case this context is actually used as key value store for supplementary info (as far as I understand). https://github.com/census-instrumentation/opencensus-go/blob/0028eeaf06735bceeb492e091b29fb43ad8e5d90/stats/record.go#L154

I guess we are not using any of the tag features anywho.